web-ui/pages/index.vue

```diff
@@ -19,18 +19,33 @@ export default {
     EditorPanel,
     AppContainer,
   },
-  mounted() {
-    window.log = output => {
-      this.addToLog(output);
-    };
-  },
   data() {
     return {
       code:
-        "const a = 10;\nconst b = 20;\nconsole.log(a + b);\nconsole.log('A String');\n",
-      console: "",
+        "const a = 10;\nconst b = 20;\nconsole.log(a + b);\nconsole.log('A String');\nsdfsdf\n",
+      console: [],
     };
   },
+  mounted() {
+    if (typeof Storage !== "undefined")
+      if (localStorage.code) this.code = localStorage.code;
```

Comment:
I dislike implicit bracers as it is easy to introduce a none obvious an error when refactoring and it keeps things more consistent if the bracers are always used. You can also remove the nested if:

```js
    if (typeof Storage !== "undefined" && localStorage.code) {
      this.code = localStorage.code;
    }
```